Strings/CardName.jsonc

```diff
@@ -106,10 +106,10 @@
 
 	"IruCa（フリーIruCa）": {
 		"ja": "IruCa（フリーIruCa）",
-		"en": "IruCa (Free)",
-		"zh-Hans": "IruCa（免费）",
-		"zh-Hant": "IruCa（免費）",
-		"zh-HK": "IruCa（免費）"
+		"en": "IruCa (For All)",
```

Comment:
I think the English should remain 'Free', though I can't find any official English resources for IruCa so I can't confirm...

(Wikipedia says Iruca Free however)

Reply:
I also noticed that on Wikipedia, "Free" goes in front of "IruCa", i.e. "Free IruCa". Should we change that format for all the IruCa cards or continue to use the current format, "IruCa (Free)"?

Reply:
I think to keep it in line with the format of PASPY, Octopus, etc. we should leave it in parentheses.

Reply:
Okay.

Reply:
Please keep the current format.